core/package.json

```diff
@@ -2,12 +2,11 @@
   "name": "@snowbridge/core",
   "private": true,
   "engines": {
-    "node": "v18.13.0",
+    "node": "v18.12.1",
```

Comment:
@doubledup  saw this warning message when initializing nix env seems node in nix is v18.12 by default? so downgrade version here.
```
WARN: Unsupported engine: wanted: {"node":"v18.13.0"} (current: {"node":"v18.12.1","pnpm":"7.14.2"})
```

Reply:
Whoops, that's right. Should be 18.12.1: https://search.nixos.org/packages?channel=22.11&show=nodejs-18_x&from=0&size=50&sort=relevance&type=packages&query=nodejs-18